src/test/script_tests.cpp

```diff
@@ -1368,6 +1368,13 @@ static CScript ScriptFromHex(const std::string& str)
     return ToScript(*Assert(TryParseHex(str)));
 }
 
+BOOST_AUTO_TEST_CASE(script_hex_conversion)
```

Comment:
nit: Might call it something like `script_byte_array_u8_vector_equivalence`?

Reply:
done